packages/core-ui/styles/modal.scss

```diff
@@ -1,6 +1,87 @@
 /* @license Copyright 2023 @paritytech/polkadot-dashboard-ui authors & contributors
 SPDX-License-Identifier: Apache-2.0 */
 
+.modal-content {
+  background: var(--background-modal);
+  width: 100%;
+  height: auto;
+  overflow: hidden;
+  position: relative;
+
+  h2 {
+    margin: 1rem 0;
+
+    &.unbounded {
+      font-family: Unbounded, sans-serif;
+    }
+
+    &.title {
+      font-size: 1.35rem;
+      margin: 1.25rem 0 0;
+    }
+  }
+
+  a {
+    color: var(--network-color-primary);
+  }
+
+  .header {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    padding: 1rem 1rem 0;
+  }
+
+  .body {
+    padding: 1rem;
+  }
+
+  .action-button {
```

Comment:
This is no longer being used (moved to `ActionItem`), so `.action-button` styling can be deleted.